specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml

```diff
@@ -33,9 +33,9 @@ options:
     package-details:
       name: "@azure/arm-contoso"
   "@azure-tools/typespec-go":
-    service-dir: "sdk/resourcemanager/contoso"
-    package-dir: "armcontoso"
-    module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
+    service-dir: "{output-dir}/sdk/resourcemanager/contoso"
```

Comment:
I assume we are overriding because we need to put all the mgmt under the `resourcemanager` folder for go?

Reply:
yes.